up42/auth.py

```diff
@@ -219,7 +219,7 @@ def _request(
         data: Union[dict, list] = {},
         querystring: dict = {},
         return_text: bool = True,
-    ) -> Union[str, dict, requests.Response]:
+    ):  # Union[str, dict, requests.Response]:
```

Comment:
Not exactly sure why only the removed class inheritance triggered lots of type checks issues of the response, but had to deactivate for now. The current potential API return types are not ideal (json, text, quicklook data chunkz), this will at some point be adressed at API level.

Reply:
Was looking at the problem before and could not make sense of it. Don't think it's critical, but it would surely be good if that would be resolved some time in the future; as this repo is open source people will wonder. Maybe create a tech ticket?